src/client/sound_effect.hpp

```diff
@@ -35,7 +43,7 @@ namespace rtype::game {
         private:
             std::vector<sf::Sound> sounds; ///< Vector of sounds
             std::vector<sf::SoundBuffer> buffers; ///< Vector of sound buffers
-            sf::Music music; ///< Music of the game
+            std::shared_ptr<sf::Music> music = std::make_shared<sf::Music>(); ///< Music of the game
```

Comment:
Maybe a comment explaining how this makes the type copyable would be appropriate.